shared/string.js

```diff
@@ -0,0 +1,27 @@
+String.prototype.isEmpty = function() {
+  return this.trim() === '';
+};
+
+String.prototype.isBlankOrGitOrBit = function() {
+  return /^$|github.com|bitbucket.org/g.test(this);
+};
+
+String.prototype.toLowerHyphenated = function() {
+  return this.toLowerCase().trim().replace(/\s/g, '-');
+};
+
+String.prototype.toCapitalizeUnhyphenated = function() {
+  return this.replace(/-/g, ' ')
+    .replace(/\b\w/g, function(m) {
+      return m.toUpperCase();
+    })
+    .replace(/\s/g, '');
+};
+
+String.prototype.toUpperCaseUnhyphenated = function() {
```

Comment:
Same here.
Also why wrap just `this.replace(/\W+/g, '')` in a function? It's pretty clear what it does by reading the regex